.github/workflows/merge-main.yml

```diff
@@ -1,10 +1,9 @@
-
 name: Main Merge
 
 on:
   push:
     branches:
-      - main
```

Comment:
@Mkankala Can you explain why you changed the trigger this way? I'm not sure why we would refer to a branch called "pipeline".

Reply:
I was trying to check the trigger for pipeline branch. Just my testing purpose.